amun/swagger.yaml

```diff
@@ -195,6 +195,26 @@ definitions:
                 minLength: 1
                 description: CPU Memory requested at build time
                 example: 256Mi
+          env:
```

Comment:
I don't think we want to have it under ``env`` key (``env`` was probably used in the issue for something else and it stands for ``environment``). What about something like ``hardware``?

Reply:
ack. changed name to hardware